tripy/tripy/frontend/trace/ops/copy.py

```diff
@@ -60,9 +60,5 @@ def copy(input: "tripy.Tensor", device: "tripy.device") -> "tripy.Tensor":
         assert np.array_equal(np.from_dlpack(output), np.array([1, 2], dtype=np.float32))
         assert output.trace_tensor.producer.device.kind == "cpu"
     """
-    from tripy.frontend.trace.ops import Storage
-
-    if isinstance(input.trace_tensor.producer, Storage) and input.trace_tensor.producer.device == device:
```

Comment:
Why remove this?

Reply:
It's a little buggy, need to rework `device` attr when MLIR-TRT can allocate on host.